### ISSUE TITLE
(brainstorming - annotation layout changes suggestion)

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#114]
ideas shown:
- larger work area:-

 - trim the 'instruction text' (would probably be ok to say "annotate all: car" on the same line.. but I was sort of hoping the 'top menu' showing annotate already tells you?)

 - move the toolbar to the left side (widescreen leverage)

- Precise rectangle outlines - 2 alternative ideas shown:- 

 - transparent area fill with opaque single-pixel outline, 

 - or thick outline (more like the current) but transparent, and the precise outline drawn single pixel outline. The intent is to clearly show the precise single-pixel boundary

- 'plural' flag: default could be 'unspecified' i.e. you may annotate groups or individuals (a later quiz can confirm). The ideal is of course a specific toggle (car/cars)

<img width="838" alt="mockup" src="https://user-images.githubusercontent.com/1120754/39346837-0778e1c8-49e8-11e8-899f-0689f895de27.png">

**Comment by dobkeratops** [comment]
(regarding the Smart Annotation mode - did you consider overlaying the smart-annotation boundary over the same image? 
having said that regardless, I can think of many other uses for a 'side-by-side' mode, so it's great to have)

**Comment by bbernhard** [comment]
Many thanks for the suggestions - really like those!

At the moment I am working on the "larger work area" part. As this already is quite a big change (had to restructure some code in order to clean up the code base a bit - I really hate Javascript ;)), I think I am going to incrementally push those changes to production. 

One thing I am unsure at the moment is: the image size. On my working git branch, I tried to make the image as big as possible. While this works great for large monitors, it's not that nice of a experience on a small laptop (-> image doesn't fit anymore on a page, so you have to scroll). 

Ideally the image size should be calculated depending on the free space, but I guess this is quite hard with the framework I am currently using. So I am thinking about introducing three different modes (small/middle/large workspace), where "small" would match the current size. It's something that could be selected in the "annotation settings".

**Comment by dobkeratops** [comment]
> . While this works great for large monitors, it's not that nice of a experience on a small laptop (-> image doesn't fit anymore on a page, so you have to scroll).

yeah I can see that. i'm often using a 13" laptop. I guess the best is if you dont need scrolling when the window is maximised

> So I am thinking about introducing three different modes (small/middle/large workspace), where "small" would match the current size. It's something that could be selected in the "annotation settings".

makes sense. You might want to use a smaller window to one side whilst doing something else, who knows..

>  But it turned out, that it gets pretty confusing when there are a lot of scribbles

fair enough.
